src/common.rs

```diff
@@ -143,7 +143,7 @@ pub struct Config {
     pub mode: Mode,
 
     /// Run ignored tests
-    pub run_ignored: bool,
+    pub run_ignored: RunIgnored,
```

Comment:
Is this the right way to fix it? [`common.rs`](https://github.com/rust-lang/rust/blob/master/src/tools/compiletest/src/common.rs#L169) in rustc/compiletest has this field declared as a `bool`.